docs/interest-groups/index.html

```diff
@@ -7,7 +7,8 @@
 	<body>
 		<button id="leave-ig">Leave an Interest Group</button>
 		<script type="module">
-			import fledge from '../../dist/esm/index.js';
+			import Fledge from '../../dist/esm/index.js';
+			const fledge = new Fledge(true);
```

Comment:
is it possible to keep the pre-existing API and export an instance of this of the object instead of the class?

Reply:
I'm sure there is, but why do we need to change it?